test-requirements.txt

```diff
@@ -0,0 +1,6 @@
+coverage>=3.7.1
```

Comment:
I didn't realize any of these were test dependencies. How did you determine these were test dependencies?